### ISSUE TITLE
Mel-frequency visualization bug

### ISSUE THREAD

**Issue by bryan-pardo** [nussl/nussl#199]
<img width="719" alt="Screen Shot 2020-12-28 at 12 32 49 PM" src="https://user-images.githubusercontent.com/26678616/103241690-da8a6100-4908-11eb-9ad2-f5e55a3d1857.png">
**Describe the bug**
I tried the nussl tutorial and found the Mel-frequency spectrogram visualization is returning erroneous visualizations. I think the issue here is that the frequency bins are not being mapped to the right index numbers for Hz.  

**Steps To Reproduce**
Example:
```
## Example of mel display issue 

# Get the signal
input_file_path = nussl.efz_utils.download_audio_file('schoolboy_fascination_excerpt.wav')
signal1 = nussl.AudioSignal(input_file_path)

# calculate the stft
stft = signal1.stft()
psd = signal1.power_spectrogram_data
print(psd.shape)

# cut off the signal at 1kHz
import numpy as np

lp_stft = signal1.stft_data.copy()
lp_cutoff = 1000  # Hz
frequency_vector = signal1.freq_vector  # a vector of frequency values for each FFT bin
idx = (np.abs(frequency_vector - lp_cutoff)).argmin()  # trick to find the index of the closest value to lp_cutoff
lp_stft[idx:, :, :] = 0.0j  # every freq above lp_cutoff is 0 now
signal1_lp = signal1.make_copy_with_stft_data(lp_stft)
signal1_lp.istft()

# plot the output
#mysig = signal1
mysig = signal1_lp
import matplotlib.pyplot as plt

plt.figure(figsize=(10, 9))
plt.subplot(311)
plt.title('Linear-frequency spectrogram')
nussl.utils.visualize_spectrogram(mysig)

plt.subplot(312)
plt.title('Log-frequency spectrogram')
nussl.utils.visualize_spectrogram(mysig, y_axis='log')

plt.subplot(313)
plt.title('Mel-frequency spectrogram')
nussl.utils.visualize_spectrogram(mysig, y_axis='mel')

plt.tight_layout()
plt.show()

```

**Expected behavior**
I expect it to show me the audio cut off at 1kHz for all the spectrograms.

**What did happen**
Every spectrogram except the Mel-frequency spectrogram showed me a 1 kHz cutoff. The Mel-frequency one showed me a cutoff around 100 Hz or so.

**Software versions***
I ran this in Google CoLab

Linux-4.19.112+-x86_64-with-Ubuntu-18.04-bionic
Python 3.6.9 (default, Oct  8 2020, 12:12:24) 
[GCC 8.4.0]
NumPy 1.19.4
SciPy 1.4.1
nussl 1.1.3rc5

**Additional context**
This code is a cut&paste of the nussl tutorial code.

**Comment by pseeth** [comment]
Hmm, I'll look into this more but the code for generating the spectrogram is just from librosa. Here is the relevant underlying code:

https://github.com/nussl/nussl/blob/af7d0c50e01d107f4ef3305b89eb130d95d0a7cd/nussl/core/utils.py#L397-L418

**Comment by ethman** [comment]
Dug into this a little but, and I can confirm this is an issue with librosa. It's [this bug](https://github.com/librosa/librosa/issues/1240). I can add a hotfix to our librosa call that will mitigate this error for now.

**Comment by ethman** [comment]
FYI: This issue has been fixed in the most recent version of nussl v1.1.4. Just pip install as normal if in a new environment, or `pip install -U nussl` to force an upgrade in an existing environment.

Here's a [colab notebook](https://colab.research.google.com/drive/1HaFjiHVRh8iyqoapf6mHbeyp3YBwor4O?usp=sharing) demonstrating the fix.
